src/main/java/com/jaquadro/minecraft/storagedrawers/client/renderer/TileEntityDrawersRenderer.java

```diff
@@ -211,46 +178,24 @@ private void renderFastItem (@Nonnull ItemStack itemStack, TileEntityDrawers til
         alignRendering(matrix, side);
         moveRendering(matrix, scaleX, scaleY, moveX, moveY, moveZ);
 
-        //List<IRenderLabel> renderHandlers = StorageDrawers.renderRegistry.getRenderHandlers();
-        //for (IRenderLabel renderHandler : renderHandlers) {
-        //    renderHandler.render(tile, tile.getGroup(), slot, 0, partialTickTime);
-        //}
-
-        Consumer<MultiBufferSource> finish = (MultiBufferSource buf) -> {
-            if (buf instanceof MultiBufferSource.BufferSource)
-                ((MultiBufferSource.BufferSource) buf).endBatch();
-        };
+        matrix.translate(0, 0, 100f);
+        matrix.scale(1, -1, 1);
+        matrix.scale(16, 16, 16);
 
-        try {
```

Comment:
Historically, the downstream block/item renderer could crash.  Sometimes other mods do weird things.  Do you otherwise know that the itemRenderer.render call can never throw?

Reply:
This is the common item renderer used by anything that needs to render an item. If it crashes while trying to render some weird item, even if we catch the error it would still crash in an ItemFrame for example, since it does essentially the same render calls.
I say, unless you have specific examples of some items being super broken with drawer rendering but not anything else and they are as recent as, let's say, 1.16, we should probably move away from exception catching as a way to detect potential errors.

Reply:
The try was added because it's something that has occasionally come up.  People aren't good at reading logs and SD is in the callstack and they open the crash issue on our repo.  It's also hard to get your game uncrashed when the renderer is crashing.

Reply:
Added try-catch back in a new commit